mozregression/bisector.py

```diff
@@ -531,13 +531,8 @@ def bisect_nightlies(self, good_date, bad_date):
     def bisect_inbound(self, good_rev, bad_rev):
         self._logger.info("Getting inbound builds between %s and %s"
                           % (good_rev, bad_rev))
-        # anything within twelve hours is potentially within the range
-        # (should be a tighter but some older builds have wrong timestamps,
-        # see https://bugzilla.mozilla.org/show_bug.cgi?id=1018907 ...
-        # we can change this at some point in the future, after those builds
-        # expire)
         handler = InboundHandler(find_fix=self.options.find_fix)
-        result = self.do_bisect(handler, good_rev, bad_rev, range=60*60*12)
+        result = self.do_bisect(handler, good_rev, bad_rev)
```

Comment:
So are we still downloading a range of a few days before the revision? I really think we need to do that, otherwise we might not be able to isolate the right changeset.